oauthenticator/github.py

```diff
@@ -108,8 +116,10 @@ def _github_client_secret_changed(self, name, old, new):
     client_id_env = 'GITHUB_CLIENT_ID'
     client_secret_env = 'GITHUB_CLIENT_SECRET'
 
-    github_organization_whitelist = Set(
-        config=True, help="Automatically whitelist members of selected organizations"
+    github_organization_whitelist = Set(help="Deprecated, use `GitHubOAuthenticator.allowed_github_organizations`", config=True,)
+
+    allowed_github_organizations = Set(
```

Comment:
I've been trying to keep reduntant `github`, `bitbucket` names out of the GitHub and Bitbucket traits:

```suggestion
    allowed_organizations = Set(
```